charts/consul/templates/server-statefulset.yaml

```diff
@@ -229,6 +229,7 @@ spec:
       priorityClassName: {{ .Values.server.priorityClassName | quote }}
       {{- end }}
       initContainers:
+      {{- if .Values.server.locality }}
```

Comment:
This renders a block that looks something like (when locality is disabled)

```yaml
initContainers:
containers:
- name: consul
....
```

Does this cause syntax errors? I guess it does.

Reply:
no